awx/settings/defaults.py

```diff
@@ -441,8 +441,8 @@ def IS_TESTING(argv=None):
 # Django Caching Configuration
 CACHES = {
     'default': {
-        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-        'LOCATION': 'unix:/var/run/memcached/memcached.sock'
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': 'unix:/var/run/redis/redis.sock'
```

Comment:
Can we specify an "isolated" database i.e. `'LOCATION': 'unix:/var/run/redis/redis.sock?db=1'`

Reply:
I can put it in and give it a go